Gemfile

```diff
@@ -4,3 +4,8 @@ source "https://rubygems.org"
 gemspec
 
 gem "pry"
+
+group :test do
+  gem "simplecov"
+  gem "codeclimate-test-reporter", "1.0.0.pre.rc2"
```

Comment:
Is there an important distinction between keeping these in the gemspec as development dependencies and moving them here to a test group?